maven_plugin_version-master/pom.xml

```diff
@@ -134,12 +134,12 @@
     <master_plugin_jaxws_version>4.0.3</master_plugin_jaxws_version>
 
     <!-- https://github.com/jakartaee/jax-ws-api -->
-    <master_dependency_jaxws_api_version>3.0.1</master_dependency_jaxws_api_version>
+    <master_dependency_jaxws_api_version>4.0.2</master_dependency_jaxws_api_version>
```

Comment:
Version 4.0.0 seems to be around since 2022; is there a reason we update now from 3.0.1 to 4.0.2/we did not go to 4.0.0 in the past?

Reply:
Version 3 is the release for Jakarta EE 9, Version 4 is the release for Jakarta EE 10.
We are switching now from Jakarta EE 9 to Jakarta EE 10.